packages/installer.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,222 @@
+$ErrorActionPreference = 'Continue'
+$global:VerbosePreference = "SilentlyContinue"
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    # Gather packages to install
+    $installedPackages = (VM-Get-InstalledPackages).Name
+    $configPath = Join-Path ${Env:VM_COMMON_DIR} "packages.xml" -Resolve
+    $configXml = [xml](Get-Content $configPath)
+    $packagesToInstall = $configXml.config.packages.package.name | Where-Object { $installedPackages -notcontains $_ }
+
+    # List packages to install
+    Write-Host "[+] Packages to install:"
+    foreach ($package in $packagesToInstall) {
+        Write-Host "`t[+] $package"
+    }
+    Start-Sleep 1
+
+    # Install the packages
+    try {
+        foreach ($package in $packagesToInstall) {
+            Write-Host "[+] Installing: $package" -ForegroundColor Cyan
+            choco install "$package" -y
+            VM-Write-Log "INFO" "Package $package has been installed"
+        }
+    } catch {
+        VM-Write-Log-Exception $_
+    }
+    VM-Write-Log "INFO" "[+] All packages complete"
+
+    # Set Profile/Version specific configurations
+    VM-Write-Log "INFO" "[+] Beginning Windows OS VM profile configuration changes"
+    VM-Apply-Configurations $(Join-Path $Env:VM_COMMON_DIR "config.xml")
+
+    # Configure PowerShell and cmd prompts
+    VM-Configure-Prompts
+
+    # Configure PowerShell Logging
+    VM-Configure-PS-Logging
+
+    # Configure Desktop\Tools folder with a Mandiant VM icon
+    $folderPath = $Env:TOOL_LIST_DIR  
+    $iconPath = Join-Path $Env:VM_COMMON_DIR "vm.ico"
+
+    # Set the icon
+    if (Test-Path -Path $folderPath -PathType Container) {
+        # Full path to the desktop.ini file inside the folder
+        $desktopIniPath = Join-Path -Path $folderPath -ChildPath 'desktop.ini'
+        
+        # Check if desktop.ini already exists
+        if (-Not (Test-Path -Path $desktopIniPath)) {
+            # Create an empty desktop.ini if it doesn't exist
+            Set-Content -Path $desktopIniPath -Value ''
+        }
+
+        # Make the folder "system" to enable custom settings like icon change
+        attrib +s $folderPath
```

Comment:
Let's stick with he format "Start-Process" so we can use the "-wait" flag to wait for it